app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -431,19 +431,21 @@ public void onResume() {
             showNearbyCardPermissionRationale();
         });
 
-        if (store.getBoolean("displayNearbyCardView", true)) {
-            checkPermissionsAndShowNearbyCardView();
-            if (nearbyNotificationCardView.cardViewVisibilityState == NearbyNotificationCardView.CardViewVisibilityState.READY) {
-                nearbyNotificationCardView.setVisibility(View.VISIBLE);
+        if(mediaDetailPagerFragment == null){
```

Comment:
Could you please add a small comment explaining why this is necessary? Just say that the cards should only be seen on the contributions list, not in media details. Thanks!

Reply:
Added!

Reply:
Thanks for the comment!